src/cpuid.cpp

```diff
@@ -46,7 +46,7 @@ POSSIBILITY OF SUCH DAMAGE.
 #include <cstring> // for std::memset
 #endif
 
-#if TORRENT_HAS_ARM
+#if TORRENT_HAS_ARM && TORRENT_HAS_ARM_NEON
```

Comment:
true.

btw, linaro is the same you have in travis and raspberry pi is a linux based system :)